AzureOpenAICustomInferenceSkill/currently_used_skillset.json

```diff
@@ -0,0 +1,68 @@
+{
+    "@odata.context": "https://amitabh-basic-canary.search.windows.net/$metadata#skillsets/$entity",
+    "@odata.etag": "\"0x8DCEFB4EAF1115B\"",
+    "name": "ignite-demo-skillset",
+    "description": "Customized Ignite Demo Skillset using a custom skill to describe an image and then merge its text",
+    "skills": [
+      {
+        "@odata.type": "#Microsoft.Skills.Custom.WebApiSkill",
+        "name": "Image Captioning Custom Skill",
+        "description": "The skill which calls a local or deployed Azure function to describe the image",
+        "context": "/document/normalized_images/*",
+        "uri": "https://shiny-enigma-5x5rrgwgwxx3p776-7071.app.github.dev/api/custom_skill",
```

Comment:
I am deliberately pointing to a locally running endpoint to showcase that the code can be run e2e without deploying to Azure functions. It's also MUCH better to debug/test the code this way.